### ISSUE TITLE
[Future Request] Dark Mode on MacOS + Design Request on Ipad/Iphone

### ISSUE THREAD

**Issue by Ghost1299** [YACReader/yacreader#424]
Hello, 
thanks for your work! Are there plans for a dark mode in MacOS? It would look great like it did in iPad.
And also for a cleaner design, the help button on iPad and iPhone takes a lot of space at the bottom
Maybe it would be cleaner to "hide" it in the settings

Greetings

**Comment by luisangelsm** [comment]
Yes, there are plans to improve the UI on macOS and in all the desktop platforms, I need to do some serious refactoring that I started time ago.

About the Help section, yeah, I know it takes space, but even when it is so evident that it is there most people don't use it, so if I hide it I am afraid no one will find it.

**Comment by luisangelsm** [comment]
Done in YACReader 10.
